src/context.cpp

```diff
@@ -78,6 +79,11 @@ context_t::log(const std::string& source, attribute::set_t attributes) {
     return std::make_unique<logging::log_t>(*m_log, std::move(attributes));
 }
 
+signal::handler_i&
+context_t::get_signal_set() const {
+    return signal_set;
```

Comment:
In fact context do not own signal handler at all, so this is ok by logic. Anyway I will fix this.